src/lib/arch/aarch64/InstructionMetadata.cc

```diff
@@ -813,6 +816,11 @@ InstructionMetadata::InstructionMetadata(const cs_insn& insn)
       // For vector arrangment of 32-bit, post_index immediate is 4
       operands[2].imm = 4;
       break;
+    case Opcode::AArch64_LD1Onev16b: {
+      // Fix incorrect access types
+      operands[0].access = CS_AC_WRITE;
+      break;
+    }
```

Comment:
If access types are not properly set by Capstone decode, you should define them for all operands, not just the destination.